docs/use-cases/experiments.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Experimentation
+description: Learn how to experiment with A/B Tests and Multivariate Tests using Featurevisor.
+ogImage: /img/og/docs-use-cases-experimentation.png
+---
+
+Running experiments like A/B Testing and Multivariate Testing is a powerful technique in product development for continuous learning and iterating based on feedback. Featurevisor can help manage those experiments with a strong governance model in your organization. {% .lead %}
+
+## What is an A/B Test?
+
+An **A/B test**, also known as **split testing** or **bucket testing**, is a controlled experiment used to compare two or more variations of a specific feature to determine which one performs better. It is commonly used in fields like web development, marketing, user experience design, and product management.
+
+## Why run A/B Tests?
+
+The primary goal of an A/B test is to measure the impact of the variations on predefined metrics or key performance indicators (KPIs). These metrics can include conversion rates, click-through rates, engagement metrics, revenue, or any other measurable outcome relevant to the experiment.
+
+By comparing the performance of the different variants, statistical analysis is used to determine if one variant outperforms the others with statistical significance. This helps decision-makers understand which variant is likely to have a better impact on the chosen metrics.
+
+## Process of running an A/B Test
```

Comment:
Ideally you'd say that the process starts with research and identifying a customer or business problem that needs to be solved. Find a problem > Turn into a testable hypothesis

Reply:
first bullet point updated below.